window/src/os/windows/window.rs

```diff
@@ -1327,6 +1373,55 @@ fn apply_theme(hwnd: HWND) -> Option<LRESULT> {
 
         if let Some(inner) = rc_from_hwnd(hwnd) {
             let mut inner = inner.borrow_mut();
+
+            // Set Arylic or Mica system Backdrop
+            let pv_attribute = match inner.config.windows_system_backdrop {
+                SystemBackdrop::Auto => DWM_SYSTEMBACKDROP_TYPE::DWMSBT_AUTO,
+                SystemBackdrop::Disable => DWM_SYSTEMBACKDROP_TYPE::DWMSBT_NONE,
+                SystemBackdrop::Acrylic => DWM_SYSTEMBACKDROP_TYPE::DWMSBT_TRANSIENTWINDOW,
+                SystemBackdrop::Mica => DWM_SYSTEMBACKDROP_TYPE::DWMSBT_MAINWINDOW,
+                SystemBackdrop::Tabbed => DWM_SYSTEMBACKDROP_TYPE::DWMSBT_TABBEDWINDOW,
+            };
+
+            // Apply Acrylic or Mica Backdrop
+            if *IS_WIN11_22H2 {
+                DwmSetWindowAttribute(
+                    hwnd,
+                    DWMWA_SYSTEMBACKDROP_TYPE,
+                    &pv_attribute as *const _ as _,
+                    4,
+                );
+            } else if inner.config.windows_system_backdrop == SystemBackdrop::Acrylic {
+                let colour: [u8; 4] = [40, 40, 40, 1]; // acrylic doesn't like to have 0 alpha
+
+                let mut policy = ACCENT_POLICY {
+                    AccentState: ACCENT_STATE::ACCENT_ENABLE_ACRYLICBLURBEHIND as _,
+                    AccentFlags: 2,
+                    GradientColour: (colour[0] as u32)
+                        | (colour[1] as u32) << 8
+                        | (colour[2] as u32) << 16
+                        | (colour[3] as u32) << 24,
+                    AnimationId: 0,
+                };
+
+                if let Ok(user) = User32::open(std::path::Path::new("user32.dll")) {
+                    (user.SetWindowCompositionAttribute)(
+                        hwnd,
+                        &mut WINDOWCOMPOSITIONATTRIBDATA {
+                            Attrib: 0x13,
+                            pvData: &mut policy as *mut _ as _,
+                            cbData: std::mem::size_of_val(&policy),
+                        },
+                    );
+                };
+            } else if !*IS_WIN10 && !*IS_WIN11_22H2 {
+                // For build versions less than 22h2 but are
+                // still win11
+                if inner.config.windows_system_backdrop == SystemBackdrop::Mica {
+                    DwmSetWindowAttribute(hwnd, DWMWA_MICA_EFFECT, &1 as *const _ as _, 4);
+                }
```

Comment:
I think we need something like this to handle the case where the user changes the config at runtime to disable mica.  Without this, I think that once mica is enabled, it will stick forever?

```suggestion
                let mica_enabled: u32 = if inner.config.windows_system_backdrop == SystemBackdrop::Mica { 1 } else { 0};
                 DwmSetWindowAttribute(hwnd, DWMWA_MICA_EFFECT, &mica_enbabled as *const _ as _, std::mem::size_of_val(mica_enabled));
```